docs/data/date-pickers/getting-started/getting-started.md

```diff
@@ -21,96 +21,76 @@ packageName: '@mui/x-date-pickers'
-// or for date-fns
-import { AdapterDateFns } from '@mui/x-date-pickers/AdapterDateFns';
-// or for Luxon
-import { AdapterLuxon } from '@mui/x-date-pickers/AdapterLuxon';
-// or for Moment.js
-import { AdapterMoment } from '@mui/x-date-pickers/AdapterMoment';
-
-function App({ children }) {
-  return (
-    <LocalizationProvider dateAdapter={AdapterDayjs}>
-      {children}
-    </LocalizationProvider>
-  );
-}
-```
+2. **The date-library** ([Day.js](https://day.js.org/), [date-fns](https://date-fns.org/), ...) manages the date manipulation.
 
-If you use another library you should import the adapter directly from the `@date-io` package as follow.
+{{"component": "modules/components/PickersInstallationInstructions.js"}}
 
-```jsx
-import { LocalizationProvider } from '@mui/x-date-pickers';
-import AdapterJalaali from '@date-io/jalaali';
+### Why do you need a library ?
 
-function App({ children }) {
-  return (
-    <LocalizationProvider dateAdapter={AdapterJalaali}>
-      {children}
-    </LocalizationProvider>
-  );
-}
-```
+Like most pickers—the MUI Date and Time Pickers need a library to manipulate the dates.
+It will be used to format, parse, and mutate the date inside all pickers.
+
+MUI Date and Time Pickers lets you choose which library **you** want to use for the date manipulation.
+This lets you pick the library you already use in your application, without adding an extra one in your bundle.
+
+To achieve this—both `@mui/x-date-pickers` and `@mui/x-date-pickers-pro` exports a set of **adapters** which expose the date manipulation libraries under a unified api.
+
+### Choosing a date library
+
+#### Available libraries
+
+The Date and Time Pickers currently support the following date libraries:
+
+- [Day.js](https://day.js.org/)
+- [date-fns](https://date-fns.org/)
+- [Luxon](https://moment.github.io/luxon/#/)
+- [Moment.js](https://momentjs.com/)
 
 :::info
-If you are using range pickers, you can import the provided and the adapter directly from `@mui/x-date-pickers-pro`:
-
-```js
-import { LocalizationProvider } from '@mui/x-date-pickers-pro';
-import { AdapterDayjs } from '@mui/x-date-pickers-pro/AdapterDayjs';
-
-function App({ children }) {
-  return (
-    <LocalizationProvider dateAdapter={AdapterDayjs}>
-      {children}
-    </LocalizationProvider>
-  );
-}
-```
+If you are using another calendar system than the _Gregorian_ one (i.e: _Jalali_ or _Hijri_ calendars)—please refer to the [Support for other calendar systems](/x/react-date-pickers/calendar-systems/) page.
+:::
 
+#### Recommended library
+
+If you are already using one of these libraries in your application—you can keep using it for the Date and Time Pickers as well.
+This will avoid bundling two libraries.
+
+If you are starting a new project without any date manipulation outside of `@mui/x-date-pickers`—consider using `dayjs` which will have the smallest impact on the bundle size of your application.
```

Comment:
"We" is generally discouraged, but I think a recommendation like this is a good exception to the rule.

```suggestion
If you are starting a new project without any date manipulation outside of `@mui/x-date-pickers`, then we recommend using `dayjs` because it will have the smallest impact on your application's bundle size.
```